README.md

```diff
@@ -1,2 +1,19 @@
 # loft-docs
 LOFT Documentation
+
+## Vector tests
+
+### Building
+
+```
+cd examples/vectorTests
+cmake -G "<generator>" -S "" -B "build"
```

Comment:
Please add some note about generator. What is it? How can I determine, which one is mine.

Reply:
Well, it is quite a laconic note, but looks good.